src/Components/Search/Suggestions/SuggestionItem.tsx

```diff
@@ -40,12 +41,18 @@ export const FirstSuggestionItem: React.FC<SuggestionItemProps> = ({
   return (
     <SuggestionItemLink
       bg={isHighlighted ? "black5" : "white100"}
-      borderBottom="1px solid"
-      borderBottomColor="black10"
+      mt={1}
+      borderTop="1px solid"
+      borderTopColor="black10"
       onClick={handleClick}
       to={href}
     >
-      <Text variant="sm">See full results for &ldquo;{query}&rdquo;</Text>
+      <Flex alignItems="center">
+        <Text variant="sm" mr={1}>
+          See full results for &ldquo;{query}&rdquo;
```

Comment:
we don't need to use `&ldquo;` and `&rdquo;` here (Just `See full results for {query}`)

Reply:
Also i think we need to highlight query (see screenshot)

<img width="521" alt="for-sam" src="https://user-images.githubusercontent.com/3513494/225020305-6ee471cf-3f59-4c63-8d22-3b1fa7071693.png">